Make CAggs materialized only by default (#2713)
timescale/docs: use-timescale/continuous-aggregates/real-time-aggregates.md

Continuous aggregates do not include the most recent data chunk from the
underlying hypertable. Real time aggregates use the aggregated data and add the
most recent raw data to it to provide accurate and up to date results, without

> needing to aggregate data as it is being written. Starting on Timescale&nbsp;1.7 to 2.12,
> real time aggregates are enabled by default, it means when you create a continuous

Reviewer: ```suggestion
needing to aggregate data as it is being written. In Timescale versions 1.7 to 2.12,
real time aggregates are enabled by default; when you create a continuous
```

aggregate view, queries to that view include the most recent data, even if

> it has not yet been aggregated. But on Timescale&nbsp;2.13 and later real time aggregates are *NOT* enabled by default.

Reviewer: ```suggestion
it has not yet been aggregated. In Timescale&nbsp;2.13 and later real time aggregates are *NOT* enabled by default.
```

Reviewer: Also, just double checking: This is switching on 2.13, we did not release it already with 2.12?
(we should merge the docs after the release if that's true)

Author: Yep, this is material for 2.13 so it should be merged after 2.13 be released!
